### PR TITLE
Add SQLite set up docs

### LINKED ISSUE #3369: Add SQLite backend set up to documentation
### What should be done?

This should go into the 

- Quickstart guide: Add section on how to set up SQLite backend, then highlight similarities or differences with Postgres backend setup
- Introduction: Update introduction section in docs

### PR DESCRIPTION
**Pull request by Fashander**
# Description

Closes #3369.

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [ ] I added/updated integration/compatibility tests (and they pass).
- [ ] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [ ] I updated user documentation.
- [ ] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: website/docs/quickstart-guide/docker.md [hunk 1]
```diff
@@ -85,6 +85,51 @@ Find out more about:
 
 - [getting logs](../configuration/logging.md#docker-logs).
 
+### SQLite Setup with Docker Compose
+
+The following steps describe a production setup for SQLite:
```
Comment (AlekSi) [inline website/docs/quickstart-guide/docker.md:90]: I don't think we should describe it as production. For example, that would be a production without backups

File: website/docs/quickstart-guide/docker.md [hunk 2]
```diff
@@ -85,6 +85,51 @@ Find out more about:
...
+     ferretdb:
+       image: ghcr.io/ferretdb/ferretdb
+       restart: on-failure
+       ports:
+         - 27017:27017
+       volumes:
+         - ./data:/data/
+       environment:
+         - FERRETDB_HANDLER=sqlite
+         - FERRETDB_SQLITE_URL=file:/data/
```
Comment (AlekSi) [inline website/docs/quickstart-guide/docker.md:105]: Let's merge #3571 first, then we could remove that

**Review by noisersup — COMMENTED** [review]
The #3571 is merged, we're ready to remove the variable, apart from that LGTM

File: website/docs/quickstart-guide/docker.md [hunk 3]
```diff
@@ -85,6 +85,51 @@ Find out more about:
...
+
+   ```yaml
+   services:
+     ferretdb:
+       image: ghcr.io/ferretdb/ferretdb
+       restart: on-failure
+       ports:
+         - 27017:27017
+       volumes:
+         - /state/
```
Comment (AlekSi) [inline website/docs/quickstart-guide/docker.md:102]:
```suggestion
       volumes:
         - ./state:/state
```

It also should be below `environment` – just like in snipped above

File: website/docs/quickstart-guide/docker.md [hunk 4]
```diff
@@ -85,6 +85,51 @@ Find out more about:
...
+     ferretdb:
+       image: ghcr.io/ferretdb/ferretdb
+       restart: on-failure
+       ports:
+         - 27017:27017
+       volumes:
+         - ./state:/state
+       environment:
+         - FERRETDB_HANDLER=sqlite
+         - FERRETDB_SQLITE_URL=file:/state/
```
Comment (AlekSi) [inline website/docs/quickstart-guide/docker.md:105]: That line can be removed now – that's the default value

**Review by AlekSi — APPROVED** [review]

**Review by chilagrow — APPROVED** [review]
:rocket:

**Pull request merged**